crates/uv/tests/it/pip_install.rs

```diff
@@ -11563,13 +11591,38 @@ requires_python = "==3.13.*"
     ----- stdout -----
 
     ----- stderr -----
-    Prepared 1 package in [TIME]
-    Installed 1 package in [TIME]
+    Prepared 5 packages in [TIME]
+    Installed 5 packages in [TIME]
+     + anyio==4.9.0
      + blinker==1.9.0
+     + idna==3.10
+     + iniconfig==2.1.0
+     + sniffio==1.3.1
+    "
+    );
+
+    // With `--all-groups`, both `iniconfig` (default group) and `pygments` (test group) should be
+    // installed.
+    let context = new_context()?;
+    uv_snapshot!(context.filters(), context.pip_install()
+        .arg("--preview")
+        .arg("-r")
+        .arg("pylock.toml")
+        .arg("--all-groups"), @r"
```

Comment:
Gosh, I was not aware we added `-r pylock.toml --group` support. That's sort of concerning, since it doesn't match the existing `--group` semantics (i.e., not applying to other input files)